### PR TITLE
Template fixes and removals

### LINKED ISSUE #52: Typo in templates and contact info useless
### Contact Details

_No response_

### Current Behaviour

There are some typos in the issue & PR templates + the contact info is useless.

### Expected Behaviour

-

### Possible Solution

#51

### Version

0.0.1

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by ruben-iteng**
# Template fixes and removals

# Description

- Fixing some typos
- Changing some words
- Add link to COC

Fixes #52 

## PR type

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# Checklist

Please read and execute the folowing:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (if applicable)
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

#### Code of Conduct

By submitting this issue, you agree to follow our [Code of Conduct](/faebryk/faebryk/blob/main/docs/CODE_OF_CONDUCT.md):

- [x] I agree to follow this project's Code of Conduct

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# 🈸 \<title\>
+# \<Prefix: \>\<Title\>
```
Comment (IoannisP-ITENG) [inline .github/pull_request_template.md:1]: `\<Prefix\>: \<Title\>`

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -8,15 +8,15 @@ Fixes # (issue)
 
 ## PR type
```
Comment (IoannisP-ITENG) [inline .github/pull_request_template.md:9]: Can we add somewhere before here something about the prefix/title?
e.g Please ensure your PR title is following the contribution guidelines(link)

**Review by IoannisP-ITENG — CHANGES_REQUESTED** [review]
Thanks for contributing!
Looks good.
I added some note and a request.
Would be nice if you could incorporate that.

File: .github/pull_request_template.md (hunk 2 above)
Comment (ruben-iteng) [reply .github/pull_request_template.md:9]: Added extra checkbox in checklist with link to guidelines

**Review by IoannisP-ITENG — APPROVED** [review]

**Pull request merged**